### ISSUE TITLE
Package rustic is incompatible

### ISSUE THREAD

**Issue by alexandrethm** [brotzeit/rustic#331]
I am trying to get `rustic` installed
```
(use-package rustic)
```

But I keep getting this error:
```
Error (use-package): Failed to install rustic: Package ‘project-0.3.0’ is unavailable
Error (use-package): Cannot load rustic
```

Inspecting the package with `list-package` gives this info:
```
Package rustic is incompatible.

     Status: Incompatible because it depends on uninstallable packages.
    Archive: melpa
    Version: 20211127.1621
     Commit: 8985814372784a0b1e6e95c4c5c7d77e2cf8739a
    Summary: Rust development environment
   Requires: emacs-26.1, rust-mode-1.0.2, dash-2.13.0, f-0.18.2, let-alist-1.0.4, markdown-mode-2.3,
               project-0.3.0 (not available), s-1.10.0, seq-2.3, spinner-1.7.3, xterm-color-1.6
   Keywords: languages 
 Maintainer: Mozilla
     Author: Mozilla
```

It seems to be the same issue than in this [reddit post](https://www.reddit.com/r/emacs/comments/po9cfj/magit_commands_broken/) but I'm not sure how to solve it.

Sorry if it's not actually an issue with `rustic`, I am still very new to emacs 🙄

**Comment by brotzeit** [comment]
Thanks for the bug report. Which emacs version are you using ? Are you using straight.el or package.el(if you don't know, just post your emacs config) ?

**Comment by alexandrethm** [comment]
I don't know exactly what was the problem, but reinstalling from scratch (after removing the `~/.emacs.d` directory) made the issue disappear.

It might have been related to the verification issue. The "spinner-1.7.4" package could not be verified so, following your instructions, I installed gnu-elpa-keyring-update (for some reason the `gpg --homedir ~/.emacs.d/elpa/gnupg --receive-keys 066DAFCB81E42C40` command was not working, so I downloaded the .tar and manually install it).

I was also confused by the popup asking if I want to install `lsp-mode` and `rust-analyzer` when opening rust files, but I realised you actually need `cargo` and `rust-analyzer` before installing `rustic` ! Following [these steps](https://robert.kra.hn/posts/2021-02-07_rust-with-emacs/#prerequisites) worked great.

In any case, thanks for your quick answer !
